doc/release-notes/5.12.1-release-notes.md

```diff
@@ -0,0 +1,109 @@
+# Dataverse Software 5.12.1
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+## Bug Fix for "Internal Server Error" When Creating a New Remote Account
+
+Unfortunately, as of 5.11 new remote users have seen "Internal Server Error" when creating an account or checking notifications. By "remote" user we mean any user that is not creating a builtin/local account. That is, remote users log in with institutional (Shibboleth), OAuth (ORCID, GitHub, or Google) or OIDC providers.
```

Comment:
A little wordy - 2nd and 3rd sentence basically same thing. Mayve Just remove the 2nd sentence and slight ly change thirst to "Remote users are those who log in with institutional (Shibboleth), OAuth (ORCID, GitHub, or Google) or OIDC providers."

Reply:
Yes. Good idea. Fixed in b2a9756.